### ISSUE TITLE
Obviously catastrophic forgetting

### ISSUE THREAD

**Issue by ghost** [brjathu/iTAML#18]
Thanks for your work, i think your work is a milestone work in lifelong machine learning.
But i run your code, the program does not seem to be running correctly.
![image](https://user-images.githubusercontent.com/58391847/103345053-4fa59400-4acb-11eb-83bf-97bd26d6fed8.png)
And the log file is here:
session_0
![image](https://user-images.githubusercontent.com/58391847/103345097-6946db80-4acb-11eb-8ae1-d9e24c33fce8.png)
session_1
![image](https://user-images.githubusercontent.com/58391847/103345147-8ed3e500-4acb-11eb-810c-89b22b1dad27.png)
session_2
![image](https://user-images.githubusercontent.com/58391847/103345175-9e532e00-4acb-11eb-8fb4-71591dd248fc.png)
session_3
![image](https://user-images.githubusercontent.com/58391847/103345185-a612d280-4acb-11eb-9d21-b7985f997b5a.png)
session_4
![image](https://user-images.githubusercontent.com/58391847/103345192-add27700-4acb-11eb-8bf6-84ca404ecdb1.png)

The only place I modify the program is start_sess = 0
So why is the program like this？

**Comment by joeljosephjin** [comment]
Im getting the same issue.. Is there a way to solve this?

**Comment by ctm-hxds** [comment]
> Im getting the same issue.. Is there a way to solve this?

No, I tried a lot of methods and finally did not solve this problem

> Im getting the same issue.. Is there a way to solve this?

**Comment by tosroka** [comment]
The session log files contain accuracy of the base model, not the meta-model. There is `meta_test` method of learner object, which trains the meta-model using the base model and memory buffer, then the accuracy is as presented in the paper (the accuracy is printed to console after `training meta tasks` step). I'm not sure what's the point of the session files though, as they don't give information about the meta-model at all, at first it was confusing for me as well.
Edit: I was running `train_mnist.py`, maybe on other datasets it's different
